generate_index.py

```diff
@@ -91,11 +92,11 @@
 
```

Comment:
You have not handled the case where the user doesn't provide any resume URL. In that case, you should not display any "view resume" text on the webpage. See the logics for blogs and hackathon to get an idea about how to handle it

Reply:
Tried to solve it considering the addBlogs.py file.
Have created addResume.py for the same.

Now it's working with False input, but upon giving the resume link as input it's still not showing the view resume part.

Please let me know where am I going wrong.